vscode-portable.json

```diff
@@ -0,0 +1,77 @@
+{
+    "homepage": "https://code.visualstudio.com/",
+    "version": "1.25.0",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://code.visualstudio.com/License/"
+    },
+    "notes": [
+        "VSCode now supports Portable Mode! Please move the following directories:",
+        "From \"$env:USERPROFILE\\.vscode\\extensions\" to \"$env:USERPROFILE\\scoop\\persist\\vscode\\data\\extensions\"",
+        "From \"$env:APPDATA\\Code\" to \"$env:USERPROFILE\\scoop\\persist\\vscode\\data\\user-data\"",
+        "Add VSCode as a context menu option by running: '$dir\\vscode-install-context.reg'"
+    ],
+    "bin": [
+        [
+            "bin\\code.cmd",
+            "code"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "code.exe",
+            "Visual Studio Code"
+        ]
+    ],
+    "persist": [
+        "data"
+    ],
+    "post_install": [
+        "if(Test-Path(\"$dir\\vscode-install-context.reg\")) {",
+        "  $codepath = \"$dir\\Code.exe\".Replace('\\', '\\\\')",
+        "  $content = Get-Content \"$dir\\vscode-install-context.reg\"",
+        "  $content = $content.Replace('$code', $codepath)",
+        "  $content | Set-Content -Path \"$dir\\vscode-install-context.reg\"",
+        "}"
+    ],
+    "checkver": {
+        "url": "https://code.visualstudio.com/sha?build=stable",
+        "jp": "$.products[?(@.platform.os == 'win32-x64-archive')].name"
```

Comment:
Are we sure we want to key off the `.name` attribute, and not the `.version` attribute?

Reply:
`.version` is just a hash. `.productVersion` and `.name` are the same.

Reply:
`.version` is github commit, which was used to build binaries.

Reply:
Hence why it should be in the checkver. Or am I missing something?